src/statsig/index.ts

```diff
@@ -94,6 +93,10 @@ export function getFeatureGate(featureGateName: StatsigFeatureGates) {
 }
 
 export function getDefaultStatsigUser(): StatsigUser {
+  // Inlined to avoid require cycles
+  // like src/statsig/index.ts -> src/redux/store.ts -> src/redux/sagas.ts -> src/positions/saga.ts -> src/statsig/index.ts
+  // and similar
+  const { store } = require('src/redux/store')
```

Comment:
With the require cycle, [FiatExchange.test.tsx](https://github.com/valora-inc/wallet/actions/runs/5131792193/jobs/9232297783#step:9:791) was failing.